tests/plugins/ml/models/undeploy_specific_models.yaml

```diff
@@ -0,0 +1,77 @@
+$schema: ../../../../json_schemas/test_story.schema.yaml
+
+description: Test the undeployment of specific models from specific nodes.
+distributions:
+  excluded:
+    - amazon-managed
+    - amazon-serverless
+version: '>= 2.11'
+warnings:
+  multiple-paths-detected: false
+prologues:
+  - path: /_plugins/_ml/models/_register
+    id: register_model
+    method: POST
+    request:
+      payload:
+        name: huggingface/sentence-transformers/msmarco-distilbert-base-tas-b
+        version: 1.0.1
+        model_format: TORCH_SCRIPT
+    output:
+      task_id: payload.task_id
+epilogues:
+  - path: /_plugins/_ml/models/{model_id}
+    parameters:
+      model_id: ${get_completed_register_model_task.model_id}
+    method: DELETE
+    status: [200, 404]
+chapters:
+  - synopsis: Wait to get completed task.
+    id: get_completed_register_model_task
+    path: /_plugins/_ml/tasks/{task_id}
+    method: GET
+    parameters:
+      task_id: ${register_model.task_id}
+    response:
+      status: 200
+      payload:
+        state: COMPLETED
+    output:
+      model_id: payload.model_id
+    retry:
+      count: 3
+      wait: 10000
+  - synopsis: Deploy a model.
+    id: deploy_model
+    path: /_plugins/_ml/models/{model_id}/_deploy
+    method: POST
+    parameters:
+      model_id: ${get_completed_register_model_task.model_id}
+    output:
+      task_id: payload.task_id
+    response:
+      status: 200
+  - synopsis: Wait to get completed task.
+    id: get_completed_deploy_model_task
+    path: /_plugins/_ml/tasks/{task_id}
+    method: GET
+    parameters:
+      task_id: ${deploy_model.task_id}
+    response:
+      status: 200
+      payload:
+        state: COMPLETED
+    output:
+      model_id: payload.model_id
+    retry:
+      count: 3
+      wait: 10000      
+  - synopsis: Undeploy a model.
```

Comment:
Combine into `undeploy.yaml` or let's call this `undeploy_model`.

 Since we're only testing the spec, in that file undeploy the specific model then undeploy all (which should be a noop). It's totally ok, we just need to exercise the schema.